src/actions/ApiActions.js

```diff
@@ -1,17 +1,40 @@
-import { SET_REF_RESULTS, SET_REF_HEADERS, SENDING_REF_REQUEST, SET_REF_QUERY, SET_REF_ERROR_MESSAGE } from '../constants/ApiConstants';
+import { SET_REF_RESULTS, SET_REF_HEADERS, SENDING_REF_REQUEST, SET_REF_QUERY, SET_REF_ERROR_MESSAGE, SET_REF_PERIOD } from '../constants/ApiConstants';
 import apiSearch from '../utils/apiSearch';
 // import searchHistory from '../utils/searchHistory';
 
 /**
  * Get info (version/last updated) from the Node server
  */
-export function refSearch(query, filter) {
+export function refSearch(query, filter, period) {
   return (dispatch) => {
     dispatch(setErrorMessage(SET_REF_ERROR_MESSAGE, ''));
     dispatch(sendingRequest(SENDING_REF_REQUEST, true));
     dispatch(setResults(SET_REF_RESULTS, { results: [] }));
```

Comment:
There is a bug here that can be fixed, if you logout from a page using this redux store, since the results [] will not be present, an error will occur preventing a user from being logged out. (They will be logged out, in terms of the redux store but they will not be redirected to the login page.)